### DISCUSSION TITLE
What is the alternative of :where() with @tailwindcss/typography?

### DISCUSSION THREAD

**Discussion by weilinzung** [tailwindlabs/tailwindcss#7105] [Help]
Since not all the browsers are supported with `:where()`, what are the alternative ways to support older browsers?

https://developer.mozilla.org/en-US/docs/Web/CSS/:where#browser_compatibility

**Comment by northern-mindset** [comment]
Not an answer, just a comment.

Indeed, this seems to be a dead end. See also #8001 and https://github.com/facebook/docusaurus/issues/7617

Currently global coverage is around 90%: https://caniuse.com/?search=where

At the same time,`postcss-preset-env` does not provide any workaround for this selector: https://github.com/csstools/postcss-plugins/discussions/509
